CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ option(WITH_STATIC "Build Indigo static library as well as shared" OFF)
 
 # Indigo API options
 option(BUILD_INDIGO "Build indigo shared library" ON)
+option(USE_FONT_MANAGER "Build indigo render with font_face_manager" OFF)
```

Comment:
We already have BUILD_STANDALONE for that. I would suggest to use font manager in case of standalone build.

Reply:
Right now, we want to use font manager only in case of WASM.
It will be added later to standalone.